resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,81 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasConfirmedLinkWarning() {
+            return localStorage.getItem('confirm_link_warning') == 'true';
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('disable_link_warning') == 'true';
         }
     }"
-    class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
+    <a
+        href="javascript:;"
```

Comment:
would it be possible to show the link here still? because it's now impossible to know what an url links to when it's for example `View` and people could put all kinds of trash up and you won't know what you will be redirected to until you click.

Reply:
just need to catch the link part so it only shows it but doesn't redirect until the JS handles it

Reply:
Right now the best way I could find to handle that was doing the following : 
`<a  href="javascript:;{{$url}}"`

Which still display the `javascript:;https://google.com` when hovering the link, so that's a bit annoying, but the other ways weren't working at all. If I specify only the {{ $url }} the modal will open for half a second and the redirection will be made without confirmation.

Reply:
will check if there's a better approach, maybe `a` was one of the things you can't catch 🤔

Reply:
Only issue remaining is here, other issue is that it's throwing some `Uncaught SyntaxError: Unexpected end of input` in the console while opening the modal.

Reply:
Just pushed something, lmk what you think about it 🤔